src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -599,16 +599,25 @@ def utask_main(uworker_input):
   return find_fixed_range(uworker_input)
 
 
-HANDLED_ERRORS = [
-    uworker_msg_pb2.ErrorType.PROGRESSION_NO_CRASH,
-    uworker_msg_pb2.ErrorType.PROGRESSION_BUILD_SETUP_ERROR,
-    uworker_msg_pb2.ErrorType.PROGRESSION_TIMEOUT,
-    uworker_msg_pb2.ErrorType.PROGRESSION_BAD_BUILD,
-    uworker_msg_pb2.ErrorType.PROGRESSION_REVISION_LIST_ERROR,
-    uworker_msg_pb2.ErrorType.PROGRESSION_BUILD_NOT_FOUND,
-    uworker_msg_pb2.ErrorType.PROGRESSION_BAD_STATE_MIN_MAX,
-    uworker_msg_pb2.ErrorType.TESTCASE_SETUP,
-]
+_ERROR_HANDLER = uworker_handle_errors.CompositeErrorHandler({
+    uworker_msg_pb2.ErrorType.PROGRESSION_REVISION_LIST_ERROR:
```

Comment:
nit: Do you think it would be better to alphabetize the handlers?

Reply:
Sure, done.